telemetry/langfuse/config.go

```diff
@@ -29,26 +29,49 @@ func WithPublicKey(publicKey string) Option {
 	}
 }
 
-// WithHost sets the Langfuse host URL.
+// WithHost sets the Langfuse host endpoint.
+// The provided host should be in "hostname:port" format (no scheme or path).
+// For cloud.langfuse.com, use "cloud.langfuse.com:443".
+// For local development, use "localhost:3000".
+//
+// Example:
+//   WithHost("cloud.langfuse.com:443")      // Production
+//   WithHost("localhost:3000")              // Local development
 func WithHost(host string) Option {
 	return func(cfg *config) {
 		cfg.host = host
 	}
 }
 
+// WithInsecure configures the exporter to use insecure connections.
+// This should only be used for development/testing environments.
+// By default, secure connections are used.
+func WithInsecure() Option {
+	return func(cfg *config) {
+		cfg.insecure = true
+	}
+}
+
 // config holds Langfuse configuration options.
 type config struct {
 	secretKey string
 	publicKey string
 	host      string
+	insecure  bool
 }
 
 // newConfigFromEnv creates a Langfuse config from environment variables.
+// Supported environment variables:
+//   LANGFUSE_SECRET_KEY: Langfuse secret key
+//   LANGFUSE_PUBLIC_KEY: Langfuse public key
+//   LANGFUSE_HOST: Langfuse host in "hostname:port" format (e.g., "cloud.langfuse.com:443")
+//   LANGFUSE_INSECURE: Set to "true" for insecure connections (development only)
```

Comment:
please update the corresponding documentation for local development.

```bash
export LANGFUSE_INSECURE="true" # for insecure connections (development only)
```
<img width="1116" height="347" alt="image" src="https://github.com/user-attachments/assets/c4c7caf9-4525-471a-bc20-1bee797d1b25" />